src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/VSRestoreSettingsUtilities.cs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.Commands;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.ProjectModel;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    public class VSRestoreSettingsUtilities
```

Comment:
It's good that you created a utility class for these apis but would these be ever used for any other project system? aren't they very specific to .net core?